pepys_import/file/e_trac_importer.py

```diff
@@ -1,15 +1,22 @@
+import os
+
 from .importer import Importer
 from datetime import datetime
 from pepys_import.core.formats import unit_registry
 from pepys_import.utils.unit_utils import convert_absolute_angle, convert_speed
+from pepys_import.core.validators import constants
 
 
 class ETracImporter(Importer):
     name = "E-Trac Format Importer"
```

Comment:
@BarisSari - we create 3 class level variables here. We always want them, for all importers/parsers.  Isn't it _tidier_ to have an `init()` method in `Importer`, and for e_trac_importer to call that in it's `init()` method?

Again, that will prevent a _lazy_ parser maintainer from not bothering to set the values.

Reply:
Done. I kept default values for importers, yet when a new importer is implemented, it is going to be necessary to pass these 3 variables to `Importer` for `init()`:
https://github.com/debrief/pepys-import/pull/108/commits/5cd71a61ccd86c742cca0831da15fa12882bd951